xivo/reverse_lookup.py

```
@@ -37,4 +62,4 @@ def format_phone_number_e164(
 
     if parsed_number is None or not is_valid_number(parsed_number):
         return None
-    return phonenumbers.format_number(parsed_number, PhoneNumberFormat.E164)
+    return phonenumbers.format_number(parsed_number, number_format)
```

I think this hides too much information behind a `None` result. For example, "invalid" numbers are silently converted to `None`. Why? 
a formatting utility should not need to validate the number(it can still be formatted even if it is not actually a real number).

In general, I would favor letting exceptions bubble up and be handled in context instead of silencing them and converting them to None, losing information.

---

In our case, this utility was made to format incoming numbers in order to make it easier to find corresponding contacts in directories. With the intent of `If I can't format it, then I should search for the number as is`.
Because, with that intent, not being able to format a number isn't preventing the service to run, then in my opinion, it shouldn't be raising an error.